crates/cubecl-runtime/src/channel/base.rs

```diff
@@ -13,6 +13,10 @@ pub trait ComputeChannel<Server: ComputeServer>: Clone + core::fmt::Debug + Send
     /// Given bindings, returns owned resources as bytes
     fn read(&self, bindings: Vec<Binding>) -> impl Future<Output = Vec<Vec<u8>>> + Send;
 
+    /// Given bindings, returns owned resources as bytes
+    fn read_tensor(&self, bindings: Vec<TensorHandle>)
```

Comment:
rename to `read_tensor_map`?

Reply:
I think some things got mixed up. Read_tensor has nothing to do with `TensorMap` except that the alignment requirements for it make it necessary. `read_tensor` is about strided memory that's read and written to with `cuMemcpy2D` to get very efficient reads/writes without forcing contiguous layout.